corelib-schemaorg/src/main/java/eu/europeana/corelib/edm/utils/SchemaOrgUtils.java

```diff
@@ -110,7 +111,10 @@ public static void processEntity(ContextualClass entity,
             SchemaOrgUtils.processOrganization((eu.europeana.corelib.definitions.edm.entity.Organization) entity, thing);
         } else if (entity instanceof Agent) {
             SchemaOrgUtils.processAgent((Agent) entity, thing);
+        } else if (entity instanceof Timespan) {
+            SchemaOrgUtils.processTimespan((Timespan) entity, thing);
```

Comment:
please create the processThing method and use it here

Reply:
kept as agreed.